s3transfer/manager.py

```diff
@@ -27,7 +28,9 @@ def __init__(self,
                  multipart_threshold=8 * MB,
                  max_concurrency=10,
                  multipart_chunksize=8 * MB,
-                 max_queue_size=0):
+                 max_queue_size=0,
```

Comment:
The total memory usage currently should be max_queue_size * 8 KB (which is the size of the each chunk in the queue, not 8 MB). The CLI uses 1000 for the queue size. I am probably going to use that.